DepositeCalcTests/Pages/LoginPage.cs

```diff
@@ -39,4 +41,75 @@ public bool IsOpened()
             return isLoginFldPresent;
         }
     }
+
+    public class RemindPasswordView
+    {
+        private IWebDriver driver;
+        private IWebElement openButton;
+        public RemindPasswordView(IWebDriver driver, IWebElement openButton)
+        {
+            this.driver = driver;
+            this.openButton = openButton;
+        }
+
+        private IWebElement RemindPasswordForm => driver.FindElement(By.XPath("//iframe[@id='remindPasswordView']"));
+        private IWebElement CloseBtn => driver.FindElement(By.XPath("//button[text()='x']"));
+        public bool IsShown => RemindPasswordForm.Displayed;
+        private IWebElement EmailFld => driver.FindElement(By.XPath("//input[@placeholder='Email address']"));
+        private IWebElement SendBtn => driver.FindElement(By.XPath("//button[text()='Send']"));
+
+        public string ErrMessageForRemindPasswordForm
```

Comment:
This property is used only in this class, so it might be made 'private'.
```suggestion
        private string ErrorMessage
```